README.md

```diff
@@ -1,52 +1,60 @@
-style to match your website's graphics requirements. Images are seamlessly delivered through a fast CDN, and much-much
-more.
+For Go, Cloudinary provides a module for simplifying the integration even further. Take a look at the [SDK guide](https://cloudinary.com/documentation/go_integration) for full documentation or for a high-level understanding of Cloudinary, see our [getting started guide](https://cloudinary.com/documentation/how_to_integrate_cloudinary).
 
-Cloudinary offers comprehensive APIs and administration capabilities and is easy to integrate with any web application,
-existing or new.
+## Setup
 
-Cloudinary provides URL and HTTP based APIs that can be easily integrated with any Web development framework.
+### Installation
 
-For Go, Cloudinary provides a module for simplifying the integration even further.
-
-## Setup ######################################################################
-
-To install Cloudinary Go SDK, use `go get`:
+To install the Cloudinary Go SDK, use the `go get` command:
 
 ```
 go get github.com/cloudinary/cloudinary-go
 ```
 
-## Usage
-
 ### Configuration
 
-Each request for building a URL of a remote cloud resource must have the `CloudName` parameter set. Each request to our
-secure APIs (e.g., image uploads, eager sprite generation) must have the `ApiKey` and `ApiSecret` parameters set.
-See [API, URLs and access identifiers](http://cloudinary.com/documentation/api_and_access_identifiers) for more details.
+For requests to our secure APIs (e.g., image uploads, asset administration) you must have the `ApiKey` and `ApiSecret` parameters set.
+You can find your account-specific configuration credentials in the **Dashboard** page of the [account console](https://cloudinary.com/console).
 
-Setting the `CloudName`, `ApiKey` and `ApiSecret` parameters can be done by initializing the Cloudinary object, or by
+Setting your `CloudName`, `ApiKey` and `ApiSecret` parameters can be done by initializing the Cloudinary object, or by
 using the CLOUDINARY_URL environment variable / system property.
 
 The entry point of the library is the Cloudinary struct.
+
 ```go
 cld, _ := cloudinary.New()
 ```
+
 Here's an example of setting the configuration parameters programatically:
 ```go
 cld, _ := cloudinary.NewFromParams("n07t21i7", "123456789012345", "abcdeghijklmnopqrstuvwxyz12")
 ```
-Another example of setting the configuration parameters by providing the CLOUDINARY_URL value:
+
+You can also set the configuration parameters by providing the CLOUDINARY_URL value:
 ```go
 cld, _ := cloudinary.NewFromUrl("cloudinary://123456789012345:abcdeghijklmnopqrstuvwxyz12@n07t21i7")
 ```
-### Upload
+
+**Learn more**: [Go configuration](https://cloudinary.com/documentation/go_integration#configuration)
+
+
+### Update 
+
+To update the Cloudinary Go SDK to the latest version, use the `go get` command with the `-u` option:
+
+```
+go get -u github.com/cloudinary/cloudinary-go
```

Comment:
@njb90 - This isn't in the docs as far as I see?  Could be nice to add under the Installation section, no?

Reply:
Agreed. Added.